addons/account_peppol/models/res_config_settings.py

```diff
@@ -277,9 +283,31 @@ def button_migrate_peppol_registration(self):
                 "Can't migrate registration with this status: %s", self.account_peppol_proxy_state
             ))
 
+        self._trigger_peppol_crons()
+
         response = self._call_peppol_proxy(
             endpoint='/api/peppol/1/migrate_peppol_registration',
             params={},
         )
         self.account_peppol_proxy_state = 'canceled'
         self.account_peppol_migration_key = response['migration_key']
+
+    def button_deregister_peppol_participant(self):
+        """
+        Deregister the edi user from Peppol network
+        """
+        self.ensure_one()
+
+        if self.account_peppol_proxy_state != 'active':
+            raise UserError(_(
+                "Can't deregister with this status: %s", self.account_peppol_proxy_state
+            ))
+
+        self._trigger_peppol_crons()
+
+        self._call_peppol_proxy(
+            endpoint='/api/peppol/1/cancel_peppol_registration',
+            params={},
```

Comment:
It looks like `params` is an optional parameter but it's not actually... To me, using kwargs here is confusing since it's not optional parameters. To me, this is better and shorter:
```suggestion
            self._call_peppol_proxy('/api/peppol/1/cancel_peppol_registration')
```
(you just need to make `params` optional)